docker.BUILD.bazel

```diff
@@ -0,0 +1,472 @@
+load("@bazel-orfs//:cc.bzl", "cc_import_binary", "cc_import_library")
+load("@bazel-orfs//:copytree.bzl", "copytree")
+load("@bazel-orfs//:openroad.bzl", "pdk")
+load("@bazel-orfs//:stringpatch.bzl", "stringpatch_binary")
+
+filegroup(
+    name = "klayout_pymod",
+    srcs = glob([
+        "lib/klayout/pymod/**/*",
+    ]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "mk",
+    srcs = glob([
+        "flow/util/utils.mk",
+        "flow/util/*.pl",
+        "flow/util/*.py",
+        "flow/scripts/*.py",
+        "flow/scripts/*.script",
+        "flow/scripts/*.tcl",
+    ]),
+)
+
+filegroup(
+    name = "makefile",
+    srcs = ["flow/Makefile"],
+    data = [":mk"],
+    visibility = ["//visibility:public"],
+)
+
+pdk(
+    name = "asap7",
+    srcs = glob([
+        "flow/platforms/asap7/**/*.gds",
+        "flow/platforms/asap7/**/*.lib.gz",
+        "flow/platforms/asap7/**/*.lef",
+        "flow/platforms/asap7/**/*.lib",
```

Comment:
This won't work for all the other PDKs. The nice thing about using a docker image is that we don't have all this whitebox stuff bleed into bazel-orfs...